baseband/vdif/header.py

```diff
@@ -681,7 +683,7 @@ def get_time(self, sample_rate=None, frame_nr=None):
         return (ref_epochs[self['ref_epoch']] +
                 TimeDelta(self['seconds'], offset, format='sec', scale='tai'))
 
-    def set_time(self, time):
+    def set_time(self, time, sample_rate=None):
```

Comment:
I agree with disliking this work-around - but reading the text of `get_time`, there is actually a case to be made for having it -- for rates about 512 Mbps, the mark5b frame number no longer is unique, so one would need a sample rate. Hence, I would argue we do here:
```
super(VDIFMark5BHeader, self).set_time(time, sample_rate=sample_rate,
frame_nr=self['frame_nr'] if sample_rate is None else None)
```

if sample_rate is None:

Reply:
p.s. Updated my comment.